content/blog/eng/technical-writers-an-introduction.md

```diff
@@ -0,0 +1,174 @@
+
+- Recover the documentation debt for old projects, when needed for refactoring and decision making.
+- Create documentation for new projects. These documentation can either be strictly technical or it can also deal with business aspects.
+- Create ready-to-use standard indexes and templates for colleagues (e.g. an index for software project documentation, a template for the documentation of microservices).
+- Provide consulting to colleagues who ask us for advice by giving them suggestions and links to tools that may be useful.
+- Review the articles on this blog from a formal and linguistic standpoint.
+- Keep up to date through courses and conferences and research tools and methods to improve our work.
+
+
+## What we do not do
+
+On the other hand, these are the activities we do **not** handle:
+
+- Create day-by-day documentation that is in charge of each team (e.g. meeting notes, minutes of retrospectives, maintenance documentation, release notes).
+- Proofreading and editing. We are available for suggestions and minor changes to documents written by others but we do not carry out review projects.
+- Create documentation for marketing/commercial communication.
+
+
+## How we plan and organize our activities
+
+Our team plans quarter-by-quarter activities, trying to align with business needs as much as possible. 
+
+However, as we said above, the company is constantly evolving and growing, and we can't deny that sudden changes have sometimes interfered with the planning of our activities. We didn't lose heart over it, though. We spent a team retrospective to figure out how we could improve our processes and anticipate needs in the best possible way. 
+
+Therefore, we created a form through which technical writing requests can be submitted and we provided instructions on how to fill it out.  Through this form it is possible to request the creation, updating, and correction of documentation. The requests received will be analyzed and discussed by the team before being taken up, put on hold or rejected. Currently, we plan to activate the form for two weeks straddling two quarters. 
+
+We chose to follow this approach because the current size of the Technical Writing team does not yet allow to allocate a technical writer to one or a few IT teams. Not being able to follow the daily activities of a team, we looked for a solution that would allow us to know the "macro-needs" of our colleagues in advance. 
+Is this the best approach ever? Probably not but it is the approach that best fits the current organization and structure of our team within the company. 
+
+
+# Conclusion
+
+Through our daily activities, we are proud to contribute to the simplification and dissemination of knowledge in the company. 
+Therefore, we hope that this article helped you learn more about our profession and made you a little curious since the acknowledgement of the technical writer's role and support from colleagues is critical to doing this job well.
```

Comment:
Choose between "is critical to do" and "is critical for doing" (https://forum.wordreference.com/threads/critical-for-or-critical-to.3018900/). 
Also, I would split this sentence as follows:

> Therefore, we hope that this article helped you learn more about our profession and made you a little curious since the acknowledgement of the technical writer's role. Remember that support from colleagues is critical to do this job well.